site/content/blogs/release-v123.en.md

```diff
@@ -41,6 +41,140 @@ Copilot v1.23 brings several new features and improvements:
 
 ## Move misplaced `http` fields in environment manifest (backward-compatible!)
 
+In [Copilot v1.23.0](https://github.com/aws/copilot-cli/releases/tag/v1.23.0), we are fixing the hierarchy
+under the `http` field in environment manifests.
```

Comment:
```suggestion
under the `http` field in environment manifest.
```

Reply:
I think plural is fine, but if singular we need a "the" before "environment"

Reply:
yuppp I am going to add "the"